src/targets/common.ini

```diff
@@ -10,6 +10,8 @@ extra_scripts =
 monitor_dtr = 0
 monitor_rts = 0
 
+mavlink_lib_dep = https://github.com/mavlink/c_library_v2.git#e54a8d2e8cf7985e689ad1c8c8f37dc0800ea87b
```

Comment:
This is a commit from Mar 7, 2024, but the repo also does not do releases.  We definitely want this pinned to something and do controlled updates.  So this is as good as anything for now..

Reply:
Yeah, there is no special thinking, it was just the latest commit when I did the PR to update the version. MAVLink as a whole does not do releases. I suspect we won't need to update often, just if we need support for a new message.

Reply:
Do you see any harm going to the current latest commit?

Reply:
Here is the compare across the 2 commits.

https://github.com/mavlink/c_library_v2/compare/e54a8d2...5575efc

Reply:
There should be no problem in jumping up to the latest commit. The actual code generation very rarely changes, is just new message definitions. Then again its probably not a good idea to change and confuse testing of this PR with extra changes if we can avoid it.